lib/server.js

```diff
@@ -39,6 +39,7 @@ var TEMPLATE = ['<script id="%s" type="application/javascript">var ',
                 Config.client.variableName,
                 ' = %s;</script>'
               ].join('');
+var TEMPLATE_REDIRECT = '<html><head><meta http-equiv="refresh" content="0;URL=\'%s\'" /></head></html>';
```

Comment:
@chunkiat82 but react-router's example seems to portray a pic that it should be treated like a redirection from the server side with 302

https://github.com/rackt/react-router/blob/master/docs/guides/advanced/ServerRendering.md